src/main/java/seedu/address/logic/parser/DukeCooksParser.java

```diff
@@ -20,6 +20,7 @@
 
     /**
      * Used for initial separation of command word and args.
+     * Ensures that commandWord is non-whitespace word, followed by any number of arguments
      */
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
```

Comment:
Function only works if commandWord is a single word (i.e addDiary, deleteDiary, addPage, deletePage)